.toys/ci.rb

```diff
@@ -265,7 +265,14 @@ def find_changed_directories files
   dirs = Set.new
   files.each do |file|
     if file =~ %r{^([^/]+)/.+$}
-      dirs << Regexp.last_match[1]
+      dir = Regexp.last_match[1]
+      dirs << dir
+      if dir =~ %r{^(.+)-v[^-]+$}
```

Comment:
This is going to catch directories like `google-cloud-vision`

Reply:
:+1: